src/pages/proposal.tsx

```diff
@@ -42,20 +41,22 @@ import './index.css'
 import './proposal.css'
 import NotFound from "decentraland-gatsby/dist/components/Layout/NotFound"
 import ProposalFooterPoi from "../components/Proposal/ProposalFooterPoi"
+import { FollowUpModal } from '../components/Modal/FollowUpModal'
 
 type ProposalPageOptions = {
   changing: boolean,
   confirmSubscription: boolean
   confirmDeletion: boolean
   confirmStatusUpdate: ProposalStatus | false
-  showVotesList: boolean
+  showVotesList: boolean,
+  showFollowUpModal: boolean
 }
 
 export default function ProposalPage() {
   const l = useFormatMessage()
   const location = useLocation()
   const params = useMemo(() => new URLSearchParams(location.search), [location.search])
-  const [options, patchOptions] = usePatchState<ProposalPageOptions>({ changing: false, confirmSubscription: false, confirmDeletion: false, confirmStatusUpdate: false, showVotesList: false })
+  const [options, patchOptions] = usePatchState<ProposalPageOptions>({ changing: false, confirmSubscription: false, confirmDeletion: false, confirmStatusUpdate: false, showVotesList: false, showFollowUpModal: params.get('newProposal') === "true" })
```

Comment:
for consistency all state derived from url should start as `false` and use a `useEffect` to detect any chance and update it

that way the update of that state came from the url change and not the `setState`